translations/German/special-terms.md

```diff
@@ -0,0 +1,275 @@
+# German translation style guide (Deutsche Übersetzungsrichtlinien)
```

Comment:
The lines in this document are very short - around 54-60 characters. Could you make them 76-80 before merging?